### ISSUE TITLE
[FEATURE REQUEST] Выбор тегов на странице новостей

### ISSUE THREAD

**Issue by TinyClayMan** [Ninja-Official/rtu-mirea-mobile#168]
**Описание предложения**
Было бы удобно, если бы была возможность выбирать несколько тегов одновременно. Это позволит с большей эффективностью искать представляющие интерес посты.

**Описание реализации**
Изменить меню тегов:
- При нажатии на тег он темнеет. Так можно выбрать несколько тегов.
- Добавить кнопки "Применить", "Выделить все" и "Очистить".
  - Кнопка "Применить" обновляет ленту и отображает те посты, которые содержат выбранные теги.
  - Кнопка "Выделить все" выделяет все теги, не обновляя ленту.
  - Кнопка "Очистить" снимает выделение тегов, не обновляя ленту.

**Comment by ValeryVerkhoturov** [comment]
На данный момент поиск по тегу имеет довольно долгую загрузку. Однако время компенсируется гарантированным ответом.
Поиск по комбинации будет иметь аналогичную O(n) и примерно такое же время, но не будет гарантировано существование статей.

**Comment by TinyClayMan** [comment]
> ... но не будет гарантировано существование статей.

Да, при добавления такого функционала стоит также в случае отсутствия статей сделать вывод надписи "Не найдено сообщений, удовлетворяющих выбранным тегам" или похожей.

**Comment by ValeryVerkhoturov** [comment]
> > ... но не будет гарантировано существование статей.
> 
> Да, при добавления такого функционала стоит также в случае отсутствия статей сделать вывод надписи "Не найдено сообщений, удовлетворяющих выбранным тегам" или похожей.

Имел ввиду, что будет потрачено время, но не будет результата.
